cmd/state-tray/main.go

```diff
@@ -44,18 +53,12 @@ func onReady() {
 }
 
 func run() error {
-	if os.Getenv("VERBOSE") == "true" {
-		// Doesn't seem to work, I think the systray lib and its logging solution is interfering
-		logging.CurrentHandler().SetVerbose(true)
-	}
+	box := packr.NewBox(assetsPath)
+	systray.SetIcon(box.Bytes(iconFile))
 
 	svcInfo := appinfo.SvcApp()
-	if !fileutils.FileExists(svcInfo.Exec()) {
-		return errs.New("Could not find: %s", svcInfo.Exec())
-	}
-
-	if _, err := exeutils.ExecuteAndForget(svcInfo.Exec(), "start"); err != nil {
-		return errs.Wrap(err, "Could not start %s", svcInfo.Exec())
+	if err := execute(svcInfo.Exec()); err != nil {
+		return errs.New("Could not execute: %s", svcInfo.Name())
```

Comment:
Nit: Please avoid doing this, it's completely unrelated to your PRs and this code is heavily in flux. so you're creating conflicts for no reason.